containers/kas/kas_app/kas_logger.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import os
+from gunicorn import glogging
+
+AUDIT_LEVEL_NUM = os.getenv("AUDIT_LEVEL_NUM", 45)
+AUDIT_ENABLED = os.getenv("AUDIT_ENABLED", "false").lower() in ("yes", "true", "t", "1")
+
+class KasLogger(glogging.Logger):
+    """Custom logger for Gunicorn log messages."""
+
+    LOG_LEVELS = {
+        "critical": logging.CRITICAL,
+        "error": logging.ERROR,
+        "warning": logging.WARNING,
+        "info": logging.INFO,
+        "debug": logging.DEBUG
+    }
+
+    LOG_LEVELS["audit"] = AUDIT_LEVEL_NUM
```

Comment:
Why not include this in the dict literal above? Is it so the iteration order is in level num order? If so you'll need to do something a little more work to init this, as it seems to be variable. But iff not:

```suggestion
    LOG_LEVELS = {
        "critical": logging.CRITICAL,
        "audit": AUDIT_LEVEL_NUM,
        "error": logging.ERROR,
        "warning": logging.WARNING,
        "info": logging.INFO,
        "debug": logging.DEBUG
    }

    LOG_LEVELS["audit"] = AUDIT_LEVEL_NUM
```